tools/standard-test/setup_limits.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
How is this script used, seems like the docker file implies some of these settings should be updated

Reply:
Will move to the README file after talking to you.